samcli/hook_packages/terraform/hooks/prepare/resources/apigw.py

```diff
@@ -11,6 +12,20 @@
     ResourceTranslationValidator,
     TFResource,
 )
+from samcli.lib.utils.resources import AWS_APIGATEWAY_METHOD
+
+LOG = logging.getLogger(__name__)
+
+INVOKE_ARN_FORMAT = (
+    "arn:aws:apigateway:${{AWS::Region}}:lambda:path/2015-03-31/functions/${{{function_logical_id}.Arn}}/invocations"
```

Comment:
aws can be also be replaced with aws partition.

```${{AWS::Partition}}```

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html